Makefile

```diff
@@ -54,3 +54,16 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+rpm:
+	mkdir -p dist
+	chmod 777 `pwd`/dist
+	cd packages && docker build -t calico-mesos-rpm-builder .
+	cd packages && docker run --name=calico-mesos-rpm-builder \
```

Comment:
Don't bother assigning a name to the build container at runtime. That way you can rerun make rpm without having to worry about existing containers iwth the same  name. That means you also will need to remove the `docker rm` step from `clean-rpm` since we no longer know the name of that container. But that's ok, we'll leave it to the user to garbage collect old containers.